src/testdata/ledger-close-meta-v0-protocol-0.json

```diff
@@ -3,24 +3,24 @@
         "v": 0,
         "v0": {
             "ledgerHeader": {
-                "hash": "6c275540e30a8b79994b45bb4a9792fea4d1a1129b017307009e15972f2fdd7f",
+                "hash": "f4ce7471b7c9b6f8be7ab97d06c7d0d832825d4b5bbbf4674b1067582d44ee49",
```

Comment:
Why did the hashes change?

Reply:
Previously, all the test setup, such as creating accounts, assets, etc, committed directly to SQL but did not add entries to the BucketList. Now we actually commit these to the BucketList, so BucketList hash and ledger header hashes have changed in all meta tests. I did an eye check, and the actual meta contents does not appear to change outside the hashes.